editor/components/post-title/index.js

```diff
@@ -116,7 +117,7 @@ class PostTitle extends Component {
 						className="editor-post-title__input"
 						value={ title }
 						onChange={ this.onChange }
-						placeholder={ __( 'Add title' ) }
+						placeholder={ placeholder || __( 'Add title' ) }
```

Comment:
Do we still need to have a default value here? What if a plugin author picks to set an empty string as a placeholder?

Reply:
I don't know :). My thinking was to make this optional in case we instantiate this editor client-side in the future.